docs/get_started.rst

```diff
@@ -1,17 +1,341 @@
-Get started
+Get Started
 ===========
 
+
+
 Install CumulusCI
 -----------------
 
+
+On macOS / Linux
+^^^^^^^^^^^^^^^^
+`Homebrew <https://brew.sh/>`_ is a prerequisite for installing CumulusCI on macOS and Linux.
+
+
+
+Install via ``pipx`` (recommended)
+*************************************
+The following will install ``pipx``, ensure that pipx is in your ``PATH`` environment variable, and then install CumulusCI:
+.. code-block:: console
```

Comment:
This code block isn't rendering correctly. You need a blank line before `..` directives.